commit 216378520bc3bd1d5ac417cfe2d58797b819110d (refs/changes/30/4630/2)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-08-03T07:34:25+00:00 (1 year, 2 months ago)
    
    TRITON-663 sdcadm fails to setup ha-binder
    Reviewed by: Dave Eddy <dave@daveeddy.com>
    Approved by: Dave Eddy <dave@daveeddy.com>

diff --git a/lib/post-setup/ha-binder.js b/lib/post-setup/ha-binder.js
index 56f8e57..9dec9e2 100644
--- a/lib/post-setup/ha-binder.js
+++ b/lib/post-setup/ha-binder.js
@@ -611,7 +611,7 @@ function do_ha_binder(subcmd, opts, args, cb) {
                 return;
             }
 
-            steps.zookeeker.updateCoreZkConfig(Object.assign({
+            steps.zookeeper.updateCoreZkConfig(Object.assign({
                 ctx: ctx
             }, commonOpts), next);
         },
